packages/frontend/src/pages/api/v1/posts.ts

```diff
@@ -113,6 +114,7 @@ const handleCreateDoxedPost = async (
     },
   });
 
+  await postTweet(post.id, address);
```

Comment:
oh we should probably only `postTweet` if in prod as well. so add a conditional here to post only if in prod